incubator/prisma/requirements.yaml

```diff
@@ -0,0 +1,5 @@
+dependencies:
+- name: postgresql
+  version: 0.15.0
```

Comment:
Should this be a version range instead of a flat version? For example, ~0.15.0 which is >= 0.15.0, < 0.16.0

Reply:
Interesting! Just added that.